game/src/core/level/utils/LevelUtils.java

```diff
@@ -16,6 +17,12 @@ public final class LevelUtils {
 
   private static final Random RANDOM = new Random();
 
+  /** These vectors can be used to calculate neighbor coordinates. */
+  private static final Coordinate[] DELTA_VECTORS =
+      new Coordinate[] {
+        new Coordinate(-1, 0), new Coordinate(1, 0), new Coordinate(0, -1), new Coordinate(0, 1),
+      };
```

Comment:
Diese Definition wird nur in `neighbours` gebraucht, oder? Dann würde ich die auch dort hineinziehen.

Reply:
Bin mit beidem nicht glücklich.
Jetzt liegt es hier so rum.....
In der Methode würde es dann aber immer wieder neu gebaut werden.... 
Java hat noch keine C-Style static in funktionen, richtig?